contracts/strategy/TokenizedStrategy.sol

```diff
@@ -234,7 +233,7 @@ contract TokenizedStrategy is ReentrancyGuard {
         // Can't mint shares to its self because of profit locking.
         require(_performanceFeeRecipient != address(this), "self");
         stor.performanceFeeRecipient = _performanceFeeRecipient;
-        // Default to a 0% performance fee.
+        // Default to a 10% performance fee.
```

Comment:
?

Reply:
I set it to 0 because we would be charging this fee twice